docs/README.md

```diff
@@ -15,12 +15,12 @@ If you're trying to learn more about Carbon, we recommend starting at
 
 ## Design
 
-The design of the Carbon language, and the rationale for that design, is
+The design of the Carbon language, and the rationale for that design, are
```

Comment:
```suggestion
Carbon language's design and rationale are
```

"are" feels awkward to me here, let's change it more. (note this will need to be reformatted)

Reply:
Cool, I also have the prettier formatter installed from the toolchain that is configured to reformat the markdown files correctly.

Reply:
@jonmeow I have updated and formatted. Please have a look.